src/openvic-simulation/military/Unit.cpp

```diff
@@ -263,90 +263,41 @@ bool UnitManager::load_unit_file(GoodManager const& good_manager, ast::NodeCPtr
 
 				ret &= add_land_unit(key, UNIT_ARGS, LAND_ARGS);
 
 				return ret;
-			}
-			break;
-		case NAVAL:
-			{
+			} break;
+			case NAVAL: {
 				Unit::icon_t naval_icon = 0;
 				bool sail = false, transport = false, capital = false, build_overseas = false;
 				uint32_t min_port_level = 0;
 				int32_t limit_per_port = 0;
 				fixed_point_t fire_range = 0, evasion = 0, supply_consumption_score = 0, hull = 0, gun_power = 0, colonial_points = 0, torpedo_attack = 0;
 
-				ret &= add_key_map_entries(key_map,
-					"naval_icon", ONE_EXACTLY, expect_uint(assign_variable_callback(naval_icon)),
-					"sail", ZERO_OR_ONE, expect_bool(assign_variable_callback(sail)),
-					"transport", ZERO_OR_ONE, expect_bool(assign_variable_callback(transport)),
-					"capital", ZERO_OR_ONE, expect_bool(assign_variable_callback(capital)),
-					"colonial_points", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(colonial_points)),
-					"can_build_overseas", ZERO_OR_ONE, expect_bool(assign_variable_callback(build_overseas)),
-					"min_port_level", ONE_EXACTLY, expect_uint(assign_variable_callback(min_port_level)),
-					"limit_per_port", ONE_EXACTLY, expect_int(assign_variable_callback(limit_per_port)),
-					"supply_consumption_score", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(supply_consumption_score)),
-					"hull", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(hull)),
-					"gun_power", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(gun_power)),
-					"fire_range", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(fire_range)),
-					"evasion", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(evasion)),
-					"torpedo_attack", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(torpedo_attack))
-				);
+				ret &= add_key_map_entries(key_map, "naval_icon", ONE_EXACTLY, expect_uint(assign_variable_callback(naval_icon)), "sail", ZERO_OR_ONE, expect_bool(assign_variable_callback(sail)), "transport", ZERO_OR_ONE, expect_bool(assign_variable_callback(transport)), "capital", ZERO_OR_ONE, expect_bool(assign_variable_callback(capital)), "colonial_points", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(colonial_points)), "can_build_overseas", ZERO_OR_ONE, expect_bool(assign_variable_callback(build_overseas)), "min_port_level", ONE_EXACTLY, expect_uint(assign_variable_callback(min_port_level)), "limit_per_port", ONE_EXACTLY, expect_int(assign_variable_callback(limit_per_port)), "supply_consumption_score", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(supply_consumption_score)), "hull", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(hull)), "gun_power", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(gun_power)), "fire_range", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(fire_range)), "evasion", ONE_EXACTLY, expect_fixed_point(assign_variable_callback(evasion)), "torpedo_attack", ZERO_OR_ONE, expect_fixed_point(assign_variable_callback(torpedo_attack)));
```

Comment:
Could this be reverted please, this being one line is absolutely a nightmare to read.